client/hosting/hosting-features/components/hosting-features.tsx

```diff
@@ -56,38 +56,43 @@ const HostingFeatures = () => {
 			? `/hosting-config/${ siteId }`
 			: `/overview/${ siteId }`
 	);
+	const hasRedirectQueryParam = searchParams.has( 'redirect_to' );
 	const hasEnTranslation = useHasEnTranslation();
 
-	const { data: siteTransferData, refetch: refetchSiteTransferData } = useSiteTransferStatusQuery(
-		siteId || undefined
-	);
+	const { data: siteTransferData } = useSiteTransferStatusQuery( siteId || undefined, {
+		refetchInterval: 3000,
+		refetchIntervalInBackground: true,
+	} );
 	// `siteTransferData?.isTransferring` is not a fully reliable indicator by itself, which is why
 	// we also look at `siteTransferData.status`
 	const isTransferInProgress =
 		siteTransferData?.isTransferring || siteTransferData?.status === transferStates.COMPLETED;
```

Comment:
Is the behavior to not show the activation button when a transfer is in either of these states something we can or want to add a test for? `TRANSFERRING` isn't something I think we're likely to introduce a regression around down the road, but the whole aspect of `COMPLETED` doesn't mean "completed" in this instance feels fragile if someone doesn't follow the logic (cognitive logic, not code logic 😉) down the road. Even with a note, better safe than sorry?

Reply:
> Since there is the next case: transfer is not in progress, it's already completed, but we haven't received info about is_wpcom_atomic: true and the button "Activate" would be rendered for a few seconds.

Gotcha. Let's rename the variable, then 👍

Also, we could maybe change the value to something like this:

```tsx
const shouldRenderActivatingCopy  = siteTransferData?.status && siteTransferData.status !== transferStates.NONE;
```

This component won't render at all once the Atomic transfer is complete, so what we really want to know here is whether an Atomic transfer was ever initiated.